Reyes_PaulAngelo/works/script.js

```diff
@@ -0,0 +1,157 @@
+    commentsContainer.appendChild(commentElement);
+
+    comments.push({ name: name, text: text, date: date });
+  }
+
+  function sortComments(ascending) {
+    comments.sort(function (a, b) {
+      return ascending ? a.date - b.date : b.date - a.date;
+    });
+    renderComments();
+  }
+
+  function renderComments() {
+    commentsContainer.innerHTML = "";
+    comments.forEach(function (comment) {
+      const commentElement = document.createElement("p");
+      commentElement.innerHTML =
+        "Name: " +
+        comment.name +
+        "<p>Comment: " +
+        comment.text +
+        "<p><small>" +
+        comment.date.toLocaleString() +
+        "</small>";
+      commentsContainer.appendChild(commentElement);
+    });
+  }
+
+  sortAscButton.addEventListener("click", function () {
+    sortComments(true);
+    sortAscButton.blur();
+  });
+
+  sortDescButton.addEventListener("click", function () {
+    sortComments(false);
+    sortDescButton.blur();
+  });
+});
+
+function searchCountry() {
+  let countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
+    return;
+  }
+
+  fetch("https://restcountries.com/v3.1/name/" + countryName)
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Country not found");
+      }
+      return response.json();
+    })
+    .then(function (countryData) {
+      let country = countryData[0];
+      let details = `
+        <h2>Country Details - ${country.name.common}</h2>
+        <p><strong>Capital:</strong> 
+        ${country.capital ? country.capital[0] : "N/A"}</p>
+        <p><strong>Population:</strong> 
+        ${country.population ? country.population.toLocaleString() : "N/A"}</p>
+        <p><strong>Region:</strong> 
+        ${country.region ? country.region : "N/A"}</p>
+        <p><strong>Languages:</strong> 
+        ${
+          country.languages
+            ? Object.values(country.languages).join(", ")
+            : "N/A"
+        }</p>
+        <p><strong>Timezones:</strong> 
+        ${country.timezones ? country.timezones.join(", ") : "N/A"}</p>
+      `;
+      document.getElementById("countryDetails").innerHTML = details;
+
+      return fetch("https://restcountries.com/v3.1/region/" + country.region);
+    })
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Region not found");
+      }
+      return response.json();
+    })
+    .then(function (regionData) {
+      let region = regionData[0].region;
+      let sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+          <div class="same-region-country">
+            <img src="${c.flags.svg}" alt="Flag of ${c.name.common}" width="50">
+            <p>${c.name.common}</p>
+          </div>
+        `;
```

Comment:
When splitting the elements into multiple lines, make sure that the continuation line is indented.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11